README.md

```diff
@@ -522,6 +522,8 @@ A rule for interacting with Kubernetes objects.
         template with the value "value" by way of first make variable
         substitution and then expand_template replacement.
```

Comment:
Do we need to update this comment here as expand_template replacement is no longer used but instead the resolver is doing the replacement?

Reply:
Good catch, yup. I'll update it.